packages/components/psammead-consent-banner/src/__snapshots__/index.test.jsx.snap

```diff
@@ -85,56 +101,85 @@ exports[`ConsentBanner should correctly render for ltr service 1`] = `
 }
 
 .emotion-6 li+li {
-  padding-top: 0.5rem;
+  margin-top: 1rem;
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-pack: center;
+  -webkit-justify-content: center;
+  -ms-flex-pack: center;
+  justify-content: center;
 }
 
 @media (min-width:37.5rem) {
   .emotion-6 {
-    width: 18%;
-    float: right;
+    -webkit-flex-direction: row;
+    -ms-flex-direction: row;
+    flex-direction: row;
+    -webkit-box-pack: justify;
+    -webkit-justify-content: space-between;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+  }
+
+  .emotion-6 li+li {
+    margin-top: 0;
   }
 }
 
+.emotion-8 {
+  text-align: center;
+  width: 100%;
+  word-break: break-word;
+}
+
 .emotion-8 button {
-  font-size: 1.125rem;
-  line-height: 1.375rem;
-  color: #F6A21D;
-  font-weight: 700;
-  background: none;
+  font-size: 0.9375rem;
+  line-height: 1.125rem;
+  width: 100%;
+  min-height: 2.75rem;
+  color: #222222;
+  font-weight: bold;
+  background-color: #FDFDFD;
   border: none;
```

Comment:
Missing a transparent border for high contrast mode on windows I believe? See last AAC on the issue

Reply:
Also missing a transparent border for the top of the component so you can see where the component starts in high contrast mode on windows and when the user changes colours in firefox?

Reply:
For a transparent border, would `solid 1px rgba(0,0,0,0)` be sufficient?

Reply:
This is what we use on the footer, would be good to be consistent 0.0625rem solid transparent

Reply:
Something similar should work for the button too, though you'll need to check that.

Reply:
Perfect, thanks